bin/jsindex

```diff
@@ -51,12 +51,12 @@ if (empty($src)) {
 
 $indexer = new \cebe\jssearch\Indexer();
 
-$files = [];
 foreach($src as $dir) {
-	$files = array_merge($files, findFiles($dir));
+	echo "Processing $dir\n";
+	$files = findFiles($dir);
 
 	if (empty($files)) {
-		error("No files where found in $dir.");
+		echo("No files where found in $dir.");
```

Comment:
echo should include a newline, also no need for parenthesis: `echo "No files where found in $dir.\n";`